engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -182,6 +188,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   when(WorkflowExecutionInProgressState) {
     case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
+      sender() ! PoisonPill
```

Comment:
Uh hmm, I almost I agree with that; if we start sending `PoisonPill`s we might start to see code that relies on that pattern.  I think it would be preferable to require the offending actors to poison themselves.